base_custom_info/models/custom_info_value.py

```diff
@@ -125,18 +128,6 @@ def _compute_value(self):
             else:
                 s.value = getattr(s, s.field_name, False)
 
-    @api.multi
-    def _inverse_value(self):
```

Comment:
I think this one is a very valuable feature. Why do you think it's not worth it anymore? :thinking:

Reply:
Because we are using widgets with the specified field. In order to make it work properly we cannot use a Char field on a Date widget. We could try to keep the function, but I don't know if that will work properly.